core/lib/generators/solidus/update/update_generator.rb

```diff
@@ -57,6 +57,11 @@ def create_new_defaults_initializer
                File.join(options[:initializer_directory], "#{options[:initializer_basename]}.rb")
     end
 
+    def install_migrations
+      say_status :copying, "migrations"
+      rake 'spree:install:migrations'
```

Comment:
Using `spree:` as the namespace is even better than using the `FROM` env variable :clap: :clap: